domains/eddy.is-cool.dev.json

```diff
@@ -0,0 +1,16 @@
+{
+    "domain": "is-cool.dev",
+    "subdomain": "eddy",
+
+    "owner": {
+        "email": "thanos22@gmx.us"
+    },
+
+    "records": {
+        "A": "34.120.54.55",
+        "AAAA": "2600:1901:0:6d85::",
+        "CNAME": "b15ff4144ad2dc44015fa8e2._acme.deno.dev."
```

Comment:
CNAME records cannot be present with other records. 

A and AAAA records can stay in one place together, but CNAME must be only and nothing else.